src/modelfree/configs/ray/aws-noisy-actions-observations.yaml

```diff
@@ -0,0 +1,103 @@
+cluster_name: aprl-cody
```

Comment:
I'd be sad if we had to add a whole new Ray config file just for this use case :(

It seems some of the differences aren't needed:
  - Docker image. Not necessary once changes are merged.
  - Instance type. Not sure why this changed TBH.
  - Checking out the `noisy_adversary_actions` branch, also not necessary once merged.

The differences I can see that are needed are:
  - IamInstanceProfile. I think I'm ok adding this to the main `aws.yaml`. It does make it slightly harder to set AWS up, but I can see access to S3 being needed for a bunch of applications.
  - Syncing the data. Not OK with this being added, but maybe we can factor it out to a bash script in experiments/. Then a user who is running these experiments can just make a one-line change to the Ray config?

Reply:
I put it in its own file so we could actually have a configuration that was known to work to run these experiments, rather than e.g. having to remember somewhere that it was necessary to add the data syncing commands in order to make that particular set of experiments work. Given that this was so fiddly to get to work in this first place, it seems worrying to lose something that out-of-the-box works for that code just for the sake of non-duplication. 

Instance type was an accident to leave in, I think.

Reply:
Having to search back through Git in order to get an experiment to run seems like not what we should be aiming for, IMO. I'd be potentially fine documenting it if it's documented in a way that it's impossible to avoid seeing if you are running the experiments in question (i.e. directly referenced in the bash script somehow). It seems like probably the best MVP solution is to have a subfolder in ray configs that basically just holds the lines that need to be added, and the anchor lines that need to come before or after them, but trying to build an entire full-featured dynamic YAML merger just for this one use case seems probably like overkill?

I basically just think that anything where you're relying on "this one weird trick you need to run this experiment" living in oral history inside someone's head, that's a bad situation to be in.